packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -200,7 +200,7 @@ export default class ImageEdit implements EditorPlugin {
                     this.options &&
                     this.options.onSelectState !== undefined
                 ) {
-                    this.setEditingImage(e.selectionRangeEx.image, this.options.onSelectState);
+                    this.setEditingImage(e.selectionRangeEx.image, ImageEditOperation.Crop);
```

Comment:
why need to change old plugin? is this for testing only?

Reply:
This is for testing, have to clean